src/client/src/index.html

```diff
@@ -4,6 +4,7 @@
   <title>Reactive Trader Cloud</title>
   <meta charset="utf-8">
   <link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico"/>
+  <link href='https://fonts.googleapis.com/css?family=Rubik:300' rel='stylesheet' type='text/css'>
```

Comment:
is it possible to pull it down locally? I've noticed some issues sometimes when reloading when you have to hit the web for such resources.